stories/dooboo-ui/Select.stories.tsx

```diff
@@ -0,0 +1,278 @@
+    string | number
+  >();
+  const [isOpen, toggleSelect] = React.useState<
+    boolean
+  >(false);
+
+  return (
+    <CustomContainer style={{ justifyContent: 'center' }}>
+      <CustomSelectContainer>
+        <Select
+          opened={isOpen}
+          loading={false}
+          disabled={false}
+          showArrow={true}
+          isDarkMode={false}
+          bordered={true}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => onSelectValue(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
+          <Item value={'Item-1'}>{'Item-1'}</Item>
+          <Item value={'Item-2'}>{'Item-2'}</Item>
+          <Item value={'Item-3'}>{'Item-3'}</Item>
+        </Select>
+      </CustomSelectContainer>
+    </CustomContainer>
+  );
+};
+
+const ChangeProps = (): React.ReactElement => {
+  const [darkTheme, setDarkTheme] = React.useState<boolean>(true);
+  const [borderless, setBorder] = React.useState<boolean>(true);
+  const [showArrow, setShowArrow] = React.useState<boolean>(true);
+  const [disabled, setDisabled] = React.useState<boolean>(false);
+
+  // [Select] states
+  const [selectedValue, onSelectValue] = React.useState<string | number>();
+  const [isOpen, toggleSelect] = React.useState<boolean>(false);
+  const [isLoading, setLoading] = React.useState<boolean>(false);
+
+  const delaySelect = (selectedItem): void => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+      onSelectValue(selectedItem);
+    }, 700);
+  };
+
+  return (
+    <CustomContainer>
+      <TogglerWrapper>
+        <View style={{ alignSelf: 'center' }}>
+          <CustomText>{'Dark mode?'}</CustomText>
+          <ThemeSwitch
+            value={darkTheme}
+            onValueChange={(): void => setDarkTheme(!darkTheme)}
+          />
+          <CustomText>{'Bordered?'}</CustomText>
+          <ThemeSwitch
+            value={borderless}
+            onValueChange={(): void => setBorder(!borderless)}
+          />
+          <CustomText>{'Show default Suffix (Arrow)?'}</CustomText>
+          <ThemeSwitch
+            value={showArrow}
+            onValueChange={(): void => setShowArrow(!showArrow)}
+          />
+          <CustomText>{'disabled?'}</CustomText>
+          <ThemeSwitch
+            value={disabled}
+            onValueChange={(): void => setDisabled(!disabled)}
+          />
+        </View>
+      </TogglerWrapper>
+      <CustomSelectContainer>
+        <Select
+          opened={isOpen}
+          loading={isLoading}
+          disabled={disabled}
+          showArrow={showArrow}
+          isDarkMode={darkTheme}
+          bordered={borderless}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => delaySelect(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
```

Comment:
Are all these props needed? How about putting default props.
Also not clear what `showArrow` does here. Removing and putting it back doesn't actually change anything when testing in `ios simulator`.

Reply:
Oh sorry. `showArrow` is `true` as a default. I was confused since this was assigned to true instead of `false` in the story.

Reply:
Try to remove all the props in the `default` story which are not necessary.